data/json/items/containers.json

```diff
@@ -588,9 +607,19 @@
         "to_hit": 1,
         "material": "aluminum",
         "contains": 1,
+        "armor_data" : {
+        "covers" : ["TORSO"],
+        "coverage" : 2,
+        "material_thickness" : 1,
+       	"encumbrance" : 0,
```

Comment:
One of these spaces is not like the other, one of these spaces just doesn't belong...

You've got a tab here. No tabs!